### ISSUE TITLE
HTMX file no longer working on sprig update

### ISSUE THREAD

**Issue by tomhewitt2201** [putyourlightson/craft-sprig#305]
Hello there,

I've just updated my project from Sprig 2.5.1 to Sprig 2.6.0. I found following the update that none of my sprig components were working and on testing I've realised that the htmx file doesn't exist in my file structure any more. ie.  cpresources/17664b72/htmx.js

This isn't something I've changed and I've not messed with any settings other than to run the update so not sure if this is a craft issue, a sprig issue or a me issue. 

I've tried switching back to an older version of sprig but to no avail. I also noticed that when I went back to utilities to see the update it's dated 04/11/2024? That might be something completely unrelated that I'm simply ignorant of. 

![Screenshot 2023-04-25 at 11 28 20](https://user-images.githubusercontent.com/37849834/234250697-db0cf4e6-8271-4305-8716-39c724b77b7c.png)

I tried updating on another project on my local machine and came across the same issue. When I look at those projects on the staging site where there hasn't been any updates yet, Sprig still works without any problems. 

![Screenshot 2023-04-25 at 11 31 37](https://user-images.githubusercontent.com/37849834/234250799-0ae85404-f813-4113-8efa-8f884fc24470.png)

Craft ver - Craft Solo 4.4.7.1

Thanks

**Comment by tomhewitt2201** [comment]
As an update I've also tried completely removing craft vendor and composer lock files and creating a new composer install which hasn't fixed the issue. 

Able to get around the problem by loading in the htmx file within the component.

**Comment by bencroker** [comment]
Sprig 2.6.0 switched from loading the htmx file from a CDN to local file storage. If that is not working then you can download htmx and manually load it using a regular `<script>` tag from a specific location in file storage or from a CDN.  But I would still like to get to the bottom of the issue you are having. 

I can't explain the date, as it is correct in the changelog. Perhaps your system’s date is not set correctly?
https://github.com/putyourlightson/craft-sprig/blob/2.6.0/CHANGELOG.md

As for the 404 error, do you have page/template caching (Blitz or otherwise) set up, and if so, can you try clearing it so that the file is regenerated?

**Comment by tomhewitt2201** [comment]
![Screenshot 2023-04-25 at 14 44 16](https://user-images.githubusercontent.com/37849834/234296157-a8d05296-7b34-4f9e-8996-049db3e6036f.png)

Hi Ben, thanks for coming back to me. Not sure if the system date is the issue there. Seems very unusual. 

I don't currently have any caching on the site. It's still in the development phase on my local system though I have also experienced the issue on a development server. Again without caching. 

I've added the folder 17664b72 and the htmx.js 1.9 file manually to the system which has obviously resolved the issue for now but as you say, it's a strange one to get to the bottom of because other than the 404 error I'm not seeing any other system log errors which might direct me about why it didn't load through.
